src/resmom/mom_main.c

```diff
@@ -8515,19 +8520,21 @@ main(int argc, char *argv[])
 				rlimit.rlim_cur = MIN_STACK_LIMIT;
 				rlimit.rlim_max = MIN_STACK_LIMIT;
 				if (setrlimit(RLIMIT_STACK, &rlimit) == -1) {
+					char msgbuf[] = "Stack limit setting failed";
 					curerror = errno;
-					sprintf(log_buffer, "Stack limit setting failed");
+					sprintf(log_buffer, "%s", msgbuf);
 					log_err(curerror, __func__, log_buffer);
```

Comment:
Why not just passing msgbuf to log_err() instead of copying it into log_buffer first?